controller/workitem.go

```diff
@@ -214,9 +214,6 @@ func (c *WorkitemController) Delete(ctx *app.DeleteWorkitemContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "error deleting work item %s", ctx.ID))
 		}
-		if err := appl.WorkItemLinks().DeleteRelatedLinks(ctx, ctx.ID); err != nil {
```

Comment:
@xcoulon Sorry for the confusion about our discussion about when to delete related links earlier in the chat. I think for now it is a good idea to delete the work item links when you delete a work item. So, can you bring this change back in?